docs/non-normative/http-migration.md

```diff
@@ -29,6 +29,9 @@ updated to the stable HTTP semantic conventions, they:
 - May drop the environment variable in their next major version and emit only
   the stable HTTP and networking conventions.
 
+The OpenTelemetry Java Agent [v2.0.0](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/tag/v2.0.0) and later already defaults to the stable
```

Comment:
thanks for the clarification!

should this text belong in the javaagent docs? I believe this doc is intended for instrumentation authors and telemetry backends rather than the end-users of specific instrumentation. 

I don't mind having it here either, but then  I think we need a section for instrumentation-specific behavior that would initially contain Java instrumentation agent.

Curious what @trask thinks.

Reply:
> should this text belong in the javaagent docs?

I think that would probably be a better place, until we solve the more general questions in #534